kettle/src/main/resources/standalone/configuration/hawkular-realm.json

```diff
@@ -57,17 +57,19 @@
       "secret" : "${uuid.hawkular.accounts.backend}"
     },
     {
-      "name": "hawkular-accounts-ui",
+      "name": "hawkular-ui",
       "enabled": true,
       "bearerOnly" : false,
       "publicClient": true,
-      "secret": "${uuid.hawkular.accounts.ui}",
+      "secret": "${uuid.hawkular.ui}",
       "fullScopeAllowed" : false,
       "redirectUris": [
+        "http://localhost:8080/*",
```

Comment:
This JSON is to import the realm into Keycloak. This file is assembled at build time on Kettle and imported during Wildfly's first boot by Keycloak. The IP and Port mentioned on this list are the allowed host/port to which Keycloak will redirect back after login. So, if an application has requested the user to be redirected back to "acme.org/metrics", then "acme.org/*" needs to be on this list. Should an application ask the user to be redirected to "evil-acme.org/metrics", Keycloak will show an error message. This list can be changed at runtime directly at Keycloak, but we need a "starting point" (which is this realm.json file). 

In other words, this means:

* Docker internal IP/port is irrelevant, as what matters is what the browser will see.
* By default, we know only a few IPs/ports: localhost:8080 for Kettle, localhost:2772 for plugin development, localhost:9000 for a different javascript tool and possibly a couple of others. 
* For a production deployment, an admin will need to either change this realm.json or to use an external Keycloak server. In any case, this list needs to be fine-tuned to the environment, as localhost will never be a real production address for Hawkular.